docs/guides/aws-private-link-workspace.md

```diff
@@ -0,0 +1,246 @@
+---
+page_title: "Enable AWS PrivateLink for Databricks Workspace"
+---
+
+# Deploying pre-requisite resources and enabling PrivateLink connections (AWS Preview)
+
+-> **Private Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+
+Databricks PrivateLink support enables private connectivity between users and their Databricks workspaces and between clusters on the data plane and core services on the control plane within the Databricks workspace infrastructure. You can use Terraform to deploy the underlying cloud resources and the private access settings resources automatically, using a programmatic approach. This guide assumes you are deploying into an existing VPC and you have set up credentials and storage configurations as per prior examples, notably here.
+
+This guide uses the following variables in configurations:
+
```

Comment:
It could be good to add an architecture diagram.

Reply:
Great suggestion, I'll add diagrams in line with the hub and spoke guide.